<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>486</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>486</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modbus Settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QSpinBox" name="retriesSpinner">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>37</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="timeoutSpinner">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="suffix">
    <string> ms</string>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
   <property name="maximum">
    <number>5000</number>
   </property>
   <property name="singleStep">
    <number>20</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>16</y>
     <width>92</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Response Timeout:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>88</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of retries:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>RELAY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>INPUTS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ADC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FACTOR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PWM</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit7">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit10">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit11">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit12">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit13">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit14">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit15">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit16">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit17">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit18">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit19">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit20">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit21">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit22">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit23">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit29">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit28">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit30">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit26">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit31">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit32">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit27">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit24">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit25">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>default</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
